core/dbt/include/global_project/macros/schema_tests/accepted_values.sql

```diff
@@ -11,32 +7,27 @@ with all_values as (
 
     from {{ model }}
 
-),
-
-validation_errors as (
-
-    select
-        value_field
-
-    from all_values
-    where value_field not in (
-        {% for value in values -%}
-            {% if quote_values -%}
-            '{{ value }}'
-            {%- else -%}
-            {{ value }}
-            {%- endif -%}
-            {%- if not loop.last -%},{%- endif %}
-        {%- endfor %}
-    )
 )
 
-select *
-from validation_errors
+select
+    value_field
+
+from all_values
+where value_field not in (
```

Comment:
The test failures are getting trickier, but this is good! I was stumped, and then I realize that we dropped the rename above:
```suggestion
select
    {{ column_name }} as value_field

from all_values
where {{ column_name }} not in (
```